igvc_navigation/GraphSearch/include/Graph.h

```diff
@@ -292,6 +366,9 @@ class Graph
   @return list of nodes who might be affected by changed cell value
   */
   std::vector<Node> getNodesAroundCellWithConfigurationSpace(const Cell& cell);
+
+private:
+  float OccupancyThresholdUChar = 178.5f;
```

Comment:
More magic numbers

Reply:
Same as above. This value is just `occupancy_threshold *255.0f`

Reply:
I think it'd be better to have this initialized using igvc::param? So that if someone deleted the default param by accident, we'd get a warning from launching the Node

Reply:
i'm currently initializing it with `igvc::getParam`  so an exception is thrown if the parameter isn't supplied.